ts/components/leftpane/overlay/OverlayClosedGroup.tsx

```diff
@@ -114,9 +135,29 @@ export const OverlayClosedGroupV2 = () => {
   useKey('Escape', closeOverlay);
 
   const noContactsForClosedGroup = isEmpty(searchTerm) && contactsToInvite.length === 0;
-
   const disableCreateButton = isCreatingGroup || (!selectedMemberIds.length && !groupName.length);
 
+  const rowRenderer = ({ index, key, style }: ListRowProps) => {
+    const memberPubkey = contactsToInvite[index];
+
+    if (!PubKey.is05Pubkey(memberPubkey)) {
+      throw new Error('Invalid member rendered in member list');
+    }
+
+    return (
+      <div key={key} style={style}>
+        <MemberListItem
+          pubkey={memberPubkey}
+          isSelected={selectedMemberIds.includes(memberPubkey)}
+          onSelect={addMemberToSelection}
+          onUnselect={removeMemberFromSelection}
+          withBorder={false}
+          disabled={isCreatingGroup}
+        />
+      </div>
+    );
+  };
```

Comment:
this has to be live outside of the render function for performance reason, I believe.